src/equations/ideal_glm_mhd_2d.jl

```diff
@@ -28,6 +28,10 @@ function IdealGlmMhdEquations2D(gamma; initial_c_h = convert(typeof(gamma), NaN)
     IdealGlmMhdEquations2D(promote(gamma, initial_c_h)...)
 end
 
+function IdealGlmMhdEquations2D(gamma, inv_gamma_minus_one, c_h)
+    IdealGlmMhdEquations2D(gamma, c_h)
+end
```

Comment:
This is required for the interface `@reset` from Accessors.jl, correct? If so, please add a comment here.

Reply:
Yes